tools/example-showcase/src/main.rs

```diff
@@ -500,6 +506,7 @@ title = \"{}\"
 template = \"example{}.html\"
 weight = {}
 description = \"{}\"
+aliases = [\"/examples{}/{}/{}\"]
```

Comment:
Can you add a comment here explaining why this is needed?

Reply:
Is the comment I left satisfactory?